feat: set DENYLIST_URL env var for bitswap-peer

olizilla opened the pull request:
Ensure DENYLIST_URL is set in env when deploying bitswap-peer

see: https://github.com/elastic-ipfs/bitswap-peer/pull/215

License: MIT

olizilla committed:
feat: set DENYLIST_URL env var

Ensure DENYLIST_URL is set in env.

see: https://github.com/elastic-ipfs/bitswap-peer/pull/215

License: MIT
Signed-off-by: Oli Evans <oli@protocol.ai>

vasco-santos commented on helm/values.yaml:
```diff
@@ -60,5 +60,6 @@ env:
   dynamodbLinkTableV1: prod-ep-v1-blocks-cars-position
   dynamodbConfigTable: prod-ep-bitswap-config
   p2pConnectionMaxConnections: 5000
+  denylistUrl: https://denylist.dag.haus
```
Do we want to start with staging in prod?

olizilla committed:
chore: deploy denylist check to staging

see: https://github.com/elastic-ipfs/bitswap-peer/pull/215

License: MIT
Signed-off-by: Oli Evans <oli@protocol.ai>

olizilla committed:
chore: use staging denylist api in prod to start with

we suspect that adding extra load to the denylist api will have us hit rate limits on cloudflare.

so we will roll this out against staging which very little traffic but the entire denylist dataset to see how it fares against that first.

if it works well without hitting limits against the staging url, then we'll move it to prod and monitor it. if it starts to hit limits we'll have to either get them raised, or consider deploying an additional denylist api just for e-ipfs (or consider other routes like sync-ing the denylist api to a db in aws)

License: MIT
Signed-off-by: Oli Evans <oli@protocol.ai>
